third_party/docfx/templates/devsite/partials/uref/class.header.tmpl.partial

```diff
@@ -129,5 +129,7 @@
 <h2>{{__global.example}}{{#example.1}}s{{/example.1}}</h2>
 {{/example.0}}
 {{#example}}
-{{{.}}}
+<div class="codewrapper">

```

Comment:
I don't think we can change this because if I recall correctly, some .NET examples have text content that shouldn't be in code blocks. That's why I added `codeexamples` (https://github.com/googleapis/doc-templates/pull/29).

Reply:
`codeexamples` only works in namespaces, not where the Node.js examples live. Seems more straight forward to use the existing `example`. I couldn't find any .Net use of example in the goldens that wasn't `[]`.